documentation-website/Writerside/topics/Data-Types.md

```diff
@@ -279,3 +279,438 @@ Teams
     .selectAll()
     .where { stringParam("Member A") eq anyFrom(Teams.memberNames.slice(1, 4)) }
 ```
+
+## Custom Data Types
+
+If a database-specific data type is not immediately supported by Exposed, any existing and open column type class can be extended or
+a custom `ColumnType` class can be implemented to achieve the same functionality.
+
+The following examples describe different ways to customize a column type, register a column with the custom type,
+and then start using it in transactions.
+
+### Ltree type
```

Comment:
I suggest to omit "type" from the titles and use the descriptive names. So instead of "Ltree type" this would be "Hierarchical tree-like data".